### PR TITLE
Add scrollbars to some settings tabs to reduce minimum size

### PR DESCRIPTION
**Pull request by LostDragonist**
The previous minimum size was around 450 x 560px.  At a DPI scaling
of 200%, this means 900 x 1120px which extends off the typical 1080p
screen.  Why is someone using 200% scaling on a 1080p screen?  I have
no clue.

Now, the minimum size is around 450 x 410px for 900 x 820 px at 200%
scaling.  Which works for anything short of a 1376 x 768 screen at
200% scaling which I just don't really care about.

### CODE REVIEW

File: src/settingsdialoggeneral.cpp [hunk 1]
```diff
@@ -15,6 +15,10 @@ GeneralSettingsTab::GeneralSettingsTab(Settings& s, SettingsDialog& d)
   addLanguages();
   selectLanguage();
 
+  // Set scrollarea to transparent
+  ui->generalScrollArea->setStyleSheet("QScrollArea { background-color: transparent; }");
+  ui->generalScrollAreaWidgetContents->setStyleSheet("background-color: transparent;");
```
Comment (Holt59) [inline src/settingsdialoggeneral.cpp:20]: The stylesheet can be set in the `.ui`, maybe it'd be better?

Comment (LostDragonist) [reply src/settingsdialoggeneral.cpp:20]: Tried it.  Didn't work for some reason. /shrug.

Comment (Holt59) [reply src/settingsdialoggeneral.cpp:20]: Works fine here... I added this to `generalScrollArea` stylesheet:

```
#generalScrollArea { background-color: transparent; }
#generalScrollAreaWidgetContents { background-color: transparent; }
```

Comment (LostDragonist) [reply src/settingsdialoggeneral.cpp]: Did the thing

**Pull request merged**